keras_cv/losses/simclr_loss.py

```diff
@@ -12,12 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import tensorflow as tf
-from tensorflow import keras
+from keras_cv.backend import keras
+from keras_cv.backend import ops
 
 LARGE_NUM = 1e9
 
 
+def l2_normalize(x, axis):
```

Comment:
Maybe we need this in Keras Core?

Reply:
It's an interesting question. The numpy way would be `numpy.linalg.norm`, but I don't think `linalg` is covered by tfnp at all.

Reply:
Probably a reasonable thing to exist in Keras Core as either a util or a layer.
For now I figured it was a simple enough little thing to leave in-line here.